packages/keyring-snap-bridge/src/util.ts

```diff
@@ -78,3 +78,16 @@ export function throwError(message: string): never {
 export function equalsIgnoreCase(a: string, b: string): boolean {
   return a.toLowerCase() === b.toLowerCase();
 }
+
+/**
+ * Sanitize a URL.
+ *
+ * @param url - The URL to sanitize.
+ * @returns The new sanitized redirect URL.
+ */
+export function sanitizeUrl(url: string): string {
+  // For now, we only re-create the URL object which should take care of most of the sanitizing, like replacing
```

Comment:
Why "most" sanitizing? Do you have other examples where it's not sanitized correctly?

Reply:
Good question, the wording is probably not the best here.  Maybe something like:
```ts
// We use a `URL` object to detect any badly formatted URL and to normalize/sanitize them.
```

WDYT?